src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -147,6 +147,9 @@ def __init__(self, *args, **kwargs):
         self.log.info("Test.random seed = %s", seed)
         self.random = random.Random(seed)
 
+        # Avoid concatenating diff output.
+        self.maxDiff = None
```

Comment:
We will need the following as the definition of this name comes from avocado:
```suggestion
        self.maxDiff = None    # pylint: disable=invalid-name
```